src/schema/controllers/user-controller.js

```diff
@@ -1,20 +1,19 @@
 import db from '../../models'
 
-export default {
-  createUser: function createUser (args) {
-    const data = args.input
-    return db.user.create({
-      firstName: data.firstName,
-      lastName: data.lastName,
-      username: data.username,
-      password: data.password
-    })
-  },
-  deleteUser: function deleteUser (args) {
-    return db.user.destroy({
-      where: {
-        id: args.id
-      }
-    })
-  }
+export function createUser (args) {
```

Comment:
Arrow functions instead of regular ones!

Reply:
If it's not a bother, I would like to ask why use arrow functions instead of regular ones. Is it because is like standard style to use?

Reply:
Yes, it's just a convention. We try to always use arrow functions because they're more concise. Although it comes with some drawbacks that force you to use regular functions sometimes (arrow functions bind the `this` keyword to the context where they were declared, instead of the object they were called on).